app/src/main/java/com/simplemobiletools/calculator/CalculatorImpl.kt

```diff
@@ -0,0 +1,216 @@
+package com.simplemobiletools.calculator
+
+import com.simplemobiletools.calculator.operation.OperationFactory
+
+class CalculatorImpl {
+    var displayedNumber: String? = null
+        private set
+    var displayedFormula: String? = null
+        private set
+    private var mLastKey: String? = null
+    private var mLastOperation: String? = null
+    private var mCallback: Calculator? = null
+
+    private var mIsFirstOperation: Boolean = false
+    private var mResetValue: Boolean = false
+    private var mBaseValue: Double = 0.0
+    private var mSecondValue: Double = 0.0
```

Comment:
remove class types from these, just make it like "mIsFirstOperation = false" etc